index.html

```diff
@@ -16,5 +16,10 @@ <h2>
       Contenido de Juanca...
     </p>
   </section>
+    <section class="section-juandavid">
+      <h2>
+        Me llamo Juan David :3
+      </h2>
+    </section>
```

Comment:
Podemos corregir la indentación de esta sección para ponerla al mismo nivel de las demás secciones.
```suggestion
  <section class="section-juandavid">
    <h2>
      Me llamo Juan David :3
    </h2>
  </section>
```